src/main/webapp/app/iris/settings/iris-settings-update/iris-common-sub-settings-update/iris-common-sub-settings-update.component.html

```diff
@@ -1,13 +1,10 @@
-<div class="form-check form-switch">
-    <input
-        class="form-check-input"
-        type="checkbox"
-        id="enabledDisabledSwitch"
-        [disabled]="!isAdmin && settingsType !== EXERCISE"
-        [(ngModel)]="enabled"
-        (change)="onEnabledChange()"
-    />
-    <label class="form-check-label" for="enabledDisabledSwitch" jhiTranslate="artemisApp.iris.settings.subSettings.enabled-disabled"> Enabled/Disabled </label>
+<div class="btn-group">
+    <div class="btn" [ngClass]="{ 'btn-success selected': enabled, 'btn-default': !enabled }" (click)="onEnable()">
+        {{ 'artemisApp.iris.settings.subSettings.enabled.on' | artemisTranslate }}
+    </div>
+    <div class="btn" [ngClass]="{ 'btn-danger selected': !enabled, 'btn-default': enabled }" (click)="onDisable()">
+        {{ 'artemisApp.iris.settings.subSettings.enabled.off' | artemisTranslate }}
+    </div>
```

Comment:
After seeing the comment by @MaximilianJG I noticed that these buttons are not disabled if the user does not have the necessary permissions. 
For this you need to add
```
[class.disabled]="!isAdmin && settingsType !== EXERCISE"
```
to both buttons.